go.mod

```diff
@@ -37,3 +37,5 @@ require (
 replace sigs.k8s.io/cluster-api-provider-aws => github.com/openshift/cluster-api-provider-aws v0.2.1-0.20201125052318-b85a18cbf338
 
 replace sigs.k8s.io/cluster-api-provider-azure => github.com/openshift/cluster-api-provider-azure v0.1.0-alpha.3.0.20201130182513-88b90230f2a4
+
+replace github.com/openshift/api => github.com/detiber/api v0.0.0-20210113181726-19e9a251beff
```

Comment:
Temporary workaround until https://github.com/openshift/api/pull/784 is merged

Reply:
+1, that makes sense for now